app/controllers/admin_controller.rb

```diff
@@ -1,3 +1,8 @@
 class AdminController < ApplicationController
-  before_filter :require_admin_auth
+  before_action :require_admin!
```

Comment:
In it's current state no.  I'll need to re-work the abilities file to handle controller scope.